weblate/addons/base.py

```diff
@@ -116,12 +116,14 @@ def configure(self, settings):
 
     def post_configure(self, run: bool = True):
         component = self.instance.component
+        project = self.instance.project
 
         # Configure events to current status
         component.log_debug("configuring events for %s add-on", self.name)
+        project.log_debug("configuring events for %s add-on", self.name)
         self.instance.configure_events(self.events)
 
-        if run:
+        if run and component:
```

Comment:
This should run on each component from `project.component_set.all()` in case of project wide.